tests/safety/test_hyundai_canfd.py

```diff
@@ -180,6 +180,27 @@ def setUp(self):
     self.safety.init_tests()
 
 
+class TestHyundaiCanfdHDA2EVAltSteering(TestHyundaiCanfdBase):
```

Comment:
We should be careful not to forget to expand this to hybrid and ICE once we see those cars with this message. Can you add a TODO comment? Ideally we design the tests in a way that makes it impossible to do this.